pom.xml

```
@@ -36,14 +36,14 @@
       <dependency>
         <groupId>com.linecorp.armeria</groupId>
         <artifactId>armeria-bom</artifactId>
-        <version>0.98.1</version>
+        <version>0.98.4</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
       <dependency>
         <groupId>io.zipkin.brave</groupId>
         <artifactId>brave-bom</artifactId>
-        <version>5.9.4</version>
+        <version>5.9.5</version>
```

might as well 5.10.1 I think?

```suggestion
        <version>5.10.1</version>
```

---

No need to wait for 5.10.1?

---

No need to wait for 0.99.0?

---

nope we don't break anything.. we'll be 5.x for the foreseeable future and won't remove any deprecated things

---

ex in sleuth, we always release latest brave, even on old branches